src/exa/devcontainer-feature.json

```
@@ -3,7 +3,7 @@
     "version": "1.0.15",
```

```suggestion
    "version": "1.0.16",
```

https://github.com/devcontainers-extra/features/actions/runs/17007114483/job/48220272174?pr=141